plugins/woocommerce-admin/client/marketing/components/CollapsibleCard/CollapsibleCard.scss

```diff
@@ -0,0 +1,15 @@
+.woocommerce-collapsible-card {
+	.components-card-header {
+		font-size: 20px;
+		font-weight: 400;
+		line-height: 28px;
+		letter-spacing: 0;
+
+		.components-button {
+			min-width: 0;
```

Comment:
Whats this `min-width: 0;` for?

Reply:
Gutenberg's Button component with icon has a min-width set to 36px. See screenshot and explanation below:

![image](https://user-images.githubusercontent.com/417342/183472739-5f2c00d9-43b5-4091-8bed-0edf5060ec31.png)

The above index.css screenshot comes from AutomateWoo CSS. When I deactivate AutomateWoo, the same CSS comes Google Listings and Ads. Eventually, I found out that the CSS comes from Gutenberg's repo:

- `.components-button.has-icon` with min-width set to `$button-size` in https://github.com/WordPress/gutenberg/blob/a4a8751a4dea46cd445258d6ee2e819ec612c482/packages/components/src/button/style.scss#L306.
- `$button-size` is set to 36px in https://github.com/WordPress/gutenberg/blob/a4a8751a4dea46cd445258d6ee2e819ec612c482/packages/base-styles/_variables.scss#L48.

In our design, we want the button to be a square 24px * 24px, but because there is a min-width 36px, the button becomes a rectangle instead of a square, so I put this `min-width: 0;` to override it.

Thinking back now, I think it may be fine to have it as a rectangle in our code base here, and we should work upstream in the gutenberg repo to fix the button min-width issue. 🤔

Reply:
I have simplified the code here, by using Button's `isSmall` prop and removing the unneeded CSS, in commit [`b22c71b` (#34080)](https://github.com/woocommerce/woocommerce/pull/34080/commits/b22c71bab9fdfcfdb162cdc5bc055f92313a652c). The button is now a 36px * 24px rectangle.